understanding/dragging-movements.html

```diff
@@ -56,213 +56,154 @@
                   
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Users who struggle with performing dragging movements can still operate an interface
-                     with a pointer interface.
+                  <li>ドラッグ動作に苦労する利用者でも、ポインタインタフェースを使用してインタフェースを操作できる。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>A map allows users to drag the view of the map around, and the map has up/down/left/right
-                     buttons to move the view as well.
+                  <li>マップでは、利用者がそのマップのビューをあらゆる方向にドラッグして移動でき、そのマップにはビューを移動するための上下左右のボタンもある。
                   </li>
                   
-                  <li>A sortable list of elements may, after tapping or clicking on a list element, provide
-                     adjacent controls for moving the element up or down in the list by simply tapping
-                     or clicking on those controls.
+                  <li>要素の並べ替え可能なリストは、リスト要素をタップ又はクリックした後、それらのコントロールをタップ又はクリックするだけで要素をリスト内で上下に移動するための隣接するコントロールを提供してもよい。
                   </li>
                   
-                  <li>A taskboard that allows users to drag and drop items between columns also provides
-                     an additional pop-up menu after tapping or clicking on items for moving the selected
-                     element to another column by tapping or clicking on pop-up menu entries.
+                  <li>利用者がカラム間で項目をドラッグ＆ドロップできるタスクボードでは、項目をタップ又はクリックした後に、ポップアップメニュー項目をタップ又はクリックして、選択した要素を別のカラムに移動するための追加のポップアップメニューも提供される。
                   </li>
                   
-                  <li>A radial control widget (color wheel) where the value can be set by dragging the marker
-                     for the currently selected color to another position, also allows picking another
-                     color value by tapping or clicking on another place in the color wheel.
+                  <li>放射状コントロールウィジェット (カラーホイール) は、現在選択されている色のマーカーを別の位置にドラッグすることで値を設定でき、カラーホイールの別の場所をタップ又はクリックして別の色の値を選択することもできる。
                   </li>
                   
-                  <li>A linear slider control widget, where the value can be set by dragging the visual
-                     indicator (thumb) showing the current value, allows tapping or clicking on any point
-                     of the slider track to change the value and set the thumb to that position. 
+                  <li>リニアスライダーコントロールウィジェットは、現在の値を示す視覚的なインジケータ (つまみ) をドラッグすることで値を設定でき、スライダートラックの任意の点をタップ又はクリックして値を変更して、つまみをその位置に設定できる。
```

Comment:
「現在の値を示す視覚的なインジケータ (つまみ) をドラッグすることで値を設定でき」ることと、「スライダートラックの任意の点をタップ又はクリックして値を変更して、つまみをその位置に設定できる」ことの関係が、並列にも因果にも読めてしまったので (原文が「**also** allows tapping or clicking on any point of the slider track to change the value and set the thumb to that position.」でないのが悪いのですが)、若干補ってみました。いかがでしょうか？

```suggestion
                  <li>リニアスライダーコントロールウィジェットは、現在の値を示す視覚的なインジケータ (つまみ) をドラッグすることで値を設定でき、また、スライダートラックの任意の点をタップ又はクリックすることでも、値を変更してつまみをその位置に設定できる。
```

Reply:
@caztcha 

語順を入れかえみてましたが、いかがでしょう…？

```suggestion
                  <li>現在の値を示す視覚的なインジケータ (つまみ) をドラッグすることで値を設定できるリニアスライダーコントロールウィジェットは、スライダートラックの任意の点をタップ又はクリックして値を変更して、つまみをその位置に設定できる。
```

Reply:
ありがとうございます！ @momdo さん案のほうが、原文に沿っていると思いますので、いただいた suggestion にてコミットします。